.travis.yml

```diff
@@ -6,4 +6,3 @@ rvm:
   - 2.2.4
   - 2.1.8
   - 2.0.0
```

Comment:
Rubocop fails for this (EOL) version. Shall we can it?

Reply:
Let's remove it.